src/metrics/deployments/__tests__/deployments.test.ts

```diff
@@ -74,7 +74,14 @@ describe("Deployments", () => {
     expect(result).toHaveLength(1);
     expect(result[0]).toMatchObject({
       output: {
-        timeSinceLastDeploy: null,
+        deployTime: 1658279953000,
+        duration: -315532800000,
```

Comment:
why is duration a negative number?
It seems that this was probably an issue already before this PR as you did not changed the logic for this part.
I will create a new ticket to investigate the issue.

Reply:
Oh, the reason is, that the `updated_at` date of the mock data is ten years before the `created_at` date. We should maybe just change that.